plugins/aea-ledger-fetchai/aea_ledger_fetchai/_cosmos.py

```diff
@@ -622,23 +599,9 @@ def get_balance(self, address: Address) -> Optional[int]:
         logger_method=_default_logger.warning,
     )
     def _try_get_balance(self, address: Address) -> Optional[int]:
-        """Try get the balance of a given account."""
-        balance = None  # type: Optional[int]
-        url = self.network_address + f"/bank/balances/{address}"
-        response = requests.get(url=url)
-        if response.status_code != 200:  # pragma: nocover
-            raise ValueError("Cannot get balance: {}".format(response.json()))
-        try:
-            result = response.json()["result"]
-            if len(result) == 0:
-                balance = 0
-            else:
-                balance = int(result[0]["amount"])
-        except KeyError:  # pragma: nocover
-            raise ValueError(
-                f"key `amount` or `result` not found in response_json={response.json()}"
-            )
-        return balance
+        bank = BankRestClient(self.rest_client)
```

Comment:
Add this in the constructors similar to `self.auth_client`, `self.wasm_client`, etc.